services/workflows-service/package.json

```
@@ -36,6 +37,7 @@
     "prepare": "npm run prisma:generate",
     "db:data-migration:generate": "plop --plopfile ./prisma/data-migrations/plopfile.js --dest ./",
     "db:data-migration:migrate": "rimraf ./dist && npm run build && nest start --entryFile ./src/data-migration/scripts/migrate",
+    "db:data-migration:sync": "nest start --entryFile ./src/data-migration/scripts/sync",
```

I think we must separate data migration and data sync into two separate concepts.
They have different goals, they are executed by different scripts.

```suggestion
    "db:data-sync": "nest start --entryFile ./src/data-migration/scripts/sync",
```